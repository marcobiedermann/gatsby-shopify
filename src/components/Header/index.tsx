import React, { ReactNode } from 'react';
import { header } from './style.module.css';

export interface HeaderProps {
  children: ReactNode;
}

function Header(props: HeaderProps) {
  return <header className={header} {...props} />;
}

export default Header;
