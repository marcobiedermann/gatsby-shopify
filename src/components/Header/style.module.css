.header {
  background-color: #fff;
  border-bottom: 1px solid #e8e8e8;
  display: flex;
  justify-content: space-between;
  padding: 1em 1.5em;
  position: sticky;
  top: 0;
}
