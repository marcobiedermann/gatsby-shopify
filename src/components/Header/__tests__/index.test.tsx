import { render } from '@testing-library/react';
import React from 'react';
import Header from '..';

describe('header component', () => {
  it('should render correctly', () => {
    const { container } = render(<Header>Header</Header>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
