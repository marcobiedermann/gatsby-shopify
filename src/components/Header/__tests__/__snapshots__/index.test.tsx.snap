// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component should render correctly 1`] = `
<header
  class="header"
>
  Header
</header>
`;
